### ISSUE TITLE
Uninstalling reWASD breaks JoyShockMapper

### ISSUE THREAD

**Issue by stack** [JibbSmart/JoyShockMapper#103]
I was originally using reWASD before I found this solution. I decided to uninstall reWASD because I didn't need it anymore. After a reboot, gyro mouse movement no longer worked. Only some times it would cause the cursor to be stuck in the top left corner. All other key mappings still worked.

After reinstalling reWASD, JoyShockMapper works properly again. I'm not sure what actually changed with the system, but everything is back to normal.

JoyShockMapper 1.6.1
Dualshock 4 via Bluetooth

**Comment by Electronicks** [comment]
Duplicate of #102

**Comment by stack** [comment]
Yes, sorry. GitHub hiccuped when submitting and created two issues.

**Comment by JibbSmart** [comment]
Have you tried connected by USB? Bluetooth support for the DS4 is still incomplete. reWASD is probably setting some settings on the controller that JSM isn't dealing with properly yet. Plan is to improve this :)

The cursor getting stuck should be fixed in the next update, which will be out real soon. For now, usually restarting the app once or twice will fix it.

**Comment by stack** [comment]
reWASD is definitely doing something to make the controller work properly, because I've only used Bluetooth for the connection and it's been flawless. I've been playing Morrowind via Bluetooth and using the gyro for mouse look and it's been pretty amazing.
